tests/common/plugins/conditional_mark/tests_mark_conditions_drop_packets.yaml

```diff
@@ -5,12 +5,12 @@
 #Hence, it is not dropped by default in Cisco-8000. For dropping link local address, it should be done through security/DATA ACL
 drop_packets/test_configurable_drop_counters.py::test_dip_link_local:
   skip:
-    reason: "MGFX topos doesn't support drop packets / Cisco 8000 platform and some mlx platforms does not drop DIP link local packets / KVM do not support drop reason in testcase."
+    reason: "M* topos doesn't support drop packets / Cisco 8000 platform and some mlx platforms does not drop DIP link local packets / KVM do not support drop reason in testcase."
```

Comment:
Need to revise this

Reply:
All testcases in drop_packets folder are for drop counter. Does M1/M2/M3 need dorp counter?

Reply:
Discussed offline, will enable these test on M1/M2/M3.